.github/workflows/gh-pages.yml

```diff
@@ -1,7 +1,7 @@
 name: GitHub Pages
 permissions:
-  contents: read
-
+  contents: write
+  deployments: write
```

Comment:
Not sure how relevant this is because we have only one job, but the [general recommendation](https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions) is

> - Set top-level permissions as `read-all` or `contents: read` as described in GitHub's [documentation](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions).
> - Set any required write permissions at the job-level. Only set the permissions required for that job; do not set `permissions: write-all` at the job level.